src/storage/mutate/UpdateEdgeProcessor.cpp

```diff
@@ -47,6 +47,10 @@ void UpdateEdgeProcessor::onProcessFinished(int32_t retNum) {
             return it->second;
         };
         for (auto& exp : returnColumnsExp_) {
+            if (!exp->prepare().ok()) {
+                LOG(ERROR) << "Expression::prepare failed";
+                return;
+            }
```

Comment:
The check here should be redundant， because `UpdateEdgeProcessor::checkAndBuildContexts` has already processed.

Reply:
Yeah, I missed that part, the result is same.